test/recipes.spec.js

```diff
@@ -333,4 +331,115 @@ describe('Recipes', () => {
         });
     });
   });
+
+  describe('Pagination', () => {
+    const url = '/api/recipes';
+    let items;
+    before(async () => {
+      const array = Array(50)
+        .fill()
+        .map(() => {
+          return chai
+            .request(server)
+            .post('/api/recipes')
+            .set({ authorization: runtimeFixture.token })
+            .send(recipe);
+        });
+      await Promise.all(array);
+    });
+
+    it('should get all recipes when no page or limit is set', done => {
+      chai
+        .request(server)
+        .get(url)
+        .end((err, res) => {
+          const { recipes } = res.body;
+          items = recipes;
```

Comment:
You shouldn't be performing setup for other examples inside a test example, as the examples in a suite should be runnable in _any_ order (or even in isolation). That's what the pre- and post-hooks are for.